frontend/src/app/_components/room-link-host/room-link-host.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { AccountService } from '@app/_services';
+import { environment } from "@environments/environment";
+
+@Component({
+  selector: 'app-room-link-host',
+  templateUrl: './room-link-host.component.html',
+  styleUrls: ['./room-link-host.component.scss']
+})
+export class RoomLinkHostComponent implements OnInit {
+  @Input() id!: string;
+  isZoomHost: boolean | undefined = false;
+  environment = environment;
+
+  constructor(private accountService: AccountService) { }
+
+  ngOnInit(): void {
+    this.isZoomHost = this.accountService.user?.zoomHost;
```

Comment:
Nit: I was going to suggest that it would be easy to make this be live with Observables, but then I realized we might have a ton of these links and each one would be a subscriber, so maybe this is better.

My only concern is whether it's possible this will get called before we've gotten a response from the me/login calls, in which case it would be out of date.

(To make it live, just do *ngIf="user$ | async as user" in an ng-container in the html and then change the ngIf on the link itself to be user?.zoomHost)

Reply:
so yes, ideally - but also "change tabs and come back" fixes that issue, i think?